src/config/kernel.md

```diff
@@ -40,7 +40,7 @@ is currently booted.
 ### Loading kernel modules at boot
 
 To load kernel modules at boot time, a `.conf` file like
-`/etc/modules-load.d/virtio.conf` can be created.
+`/etc/modules-load.d/virtio.conf` can be created with the contents.
```

Comment:
I don't think this improves clarity

Reply:
saw this had it. https://github.com/void-linux/void-docs/blob/master/src/config/media/alsa.md
since code and files are in the same block it seemed off